components/collection/unlockable/UnlockableLandingMobileBanner.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div
+    class="is-flex border-top w-full is-justify-content-space-between is-align-items-center px-4 py-3 mt-7">
+    <div class="is-flex is-align-items-center pl-1">
+      <img
+        width="42"
+        height="42"
+        src="/drop/unlockable-pulse-static.svg"
+        alt="unlockable icon" />
+      <span>{{ $t('mint.unlockable.freeDrop') }}</span>
+    </div>
+    <nuxt-link class="has-text-weight-bold pr-1" to="/ahp/drops/berlin-drop">
```

Comment:
is right padding necessary?

Reply:
removed.

At first I tried to get the 20px margin with pr-1 and the parent's px-4, but now I've found a more appropriate way: container class